lldb/packages/Python/lldbsuite/test/make/Makefile.rules

```diff
@@ -267,7 +274,9 @@ endif
 CFLAGS += $(CFLAGS_EXTRAS)
 CXXFLAGS += -std=c++11 $(CFLAGS) $(ARCH_CXXFLAGS)
 LD = $(CC)
-LDFLAGS ?= $(CFLAGS)
+# Copy common options to the linker flags (dwarf, arch. & etc).
+# Note: we get some 'garbage' options for linker here (such as -I, --isystem & etc).
+LDFLAGS += $(CFLAGS)
```

Comment:
Is there a specific reason for changing this? (I think `+=` makes more sense than `?=`, but I'm curious if it makes any difference and when)

Reply:
Okay, sounds good.